src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -61,7 +61,7 @@
     <SnappyBinaries>Core/Compression/Snappy/lib/win</SnappyBinaries>
   </PropertyGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition=" '$(OS)' == 'Windows_NT' ">
```

Comment:
Didn't the report say that it worked with 2.10 release, which already had Snappy?

Reply:
not really. The snappy with native libraries was added only in 2.11. See here: https://jira.mongodb.org/browse/CSHARP-2966

Reply:
We checked and this issue also happens with snappy. The reason why the reporter doesn't see it, mostlikely is related to the fact that `vs for mac` shows only the first thrown error